pom.xml

```diff
@@ -118,7 +119,7 @@
 		<dependency>
 			<groupId>com.esotericsoftware</groupId>
 			<artifactId>kryo</artifactId>
-			<version>4.0.0</version>
+			<version>5.0.0-RC7</version>
```

Comment:
I'm subscribed to their issue regarding releasing V5 proper - just wondering if we should wait until this is released as a proper version before releasing this component (there's a later RC anyway which we could use instead)